packages/venia-ui/package.json

```diff
@@ -82,6 +82,7 @@
     "apollo-cache-persist": "~0.1.1",
     "apollo-client": "2.6.4",
     "apollo-link-http": "~1.5.11",
+    "graphql": "~14.3.1",
```

Comment:
This was a transitive peer dependency before (via apollo-client). Now that we're using it in venia-ui/lib/util, made it explicit.